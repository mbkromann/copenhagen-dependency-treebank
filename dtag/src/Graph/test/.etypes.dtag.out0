{"adj" => ["appa", "appr", "coord", "mod", "modo", "modp", "modr", "name", "namef", "namel", "pnct", "rel", "title", "voc", "xpl", "xtop"], "comp" => ["aobj", "avobj", "dobj", "expl", "fobj", "iobj", "lobj", "nobj", "numa", "numm", "part", "pobj", "possd", "pred", "qobj", "subj", "tobj", "vobj"], "deep" => ["aobj", "avobj", "dobj", "expl", "fobj", "iobj", "lobj", "nobj", "numa", "numm", "part", "pobj", "possd", "pred", "qobj", "subj", "tobj", "vobj", "appa", "appr", "coord", "mod", "modo", "modp", "modr", "name", "namef", "namel", "pnct", "rel", "title", "voc", "xpl", "xtop", "<avobj>", "<dobj>", "<lobj>", "<nobj>", "<pobj>", "<pred>", "<subj>", "<mod>", "<xpl>", "<pobj:nobj>", "<subj:pobj>", "conj", "err", "list", "rep"], "fill" => ["[avobj]", "[dobj]", "[expl]", "[iobj]", "[mod]", "[namef]", "[nobj]", "[pobj]", "[possd]", "[pred]", "[subj]"], "gap" => ["<avobj>", "<dobj>", "<lobj>", "<nobj>", "<pobj>", "<pred>", "<subj>", "<mod>", "<xpl>", "<pobj:nobj>", "<subj:pobj>"], "land" => ["land", "xland"], "other" => ["conj", "err", "list", "rep"], "ref" => ["ref"], "surf" => ["land", "xland"]}
