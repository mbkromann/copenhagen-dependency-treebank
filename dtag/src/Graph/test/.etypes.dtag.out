PostScript files loaded from /opt/dtag/
{"adj" => [], "comp" => [], "land" => [], "other" => []}

